contracts/vaults/STETHVault/base/RibbonVault.sol

```diff
@@ -0,0 +1,650 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.3;
+pragma experimental ABIEncoderV2;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "../../../vendor/CustomSafeERC20.sol";
+import {IWSTETH} from "../../../interfaces/ISTETH.sol";
+import {IWETH} from "../../../interfaces/IWETH.sol";
+import {
+    OptionsVaultSTETHStorage
+} from "../../../storage/OptionsVaultSTETHStorage.sol";
+import {Vault} from "../../../libraries/Vault.sol";
+import {VaultLifecycleSTETH} from "../../../libraries/VaultLifecycleSTETH.sol";
+import {ShareMath} from "../../../libraries/ShareMath.sol";
+
+contract RibbonVault is OptionsVaultSTETHStorage {
+    using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+    using ShareMath for Vault.DepositReceipt;
+
+    /************************************************
+     *  IMMUTABLES & CONSTANTS
+     ***********************************************/
+
+    address public immutable USDC;
+    address public immutable WETH;
+    address public immutable LDO = 0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32;
```

Comment:
Any reason this is not provided via the constructor like the rest of the tokens?